.github/workflows/checks.yaml

```diff
@@ -180,11 +180,10 @@ jobs:
       - name: Check documentation build and links
         run: |
           . venv/bin/activate
-          cd doc
-          pre-commit run --hook-stage push sphinx-generated-doc --all-files || {
+          tox -e docs || {
```

Comment:
Are we sure this is correct? Wouldn't this always `exit 1`?

Reply:
If ``tox -e docs`` is true this exit with 0